lib/any.js

```diff
@@ -27,18 +27,17 @@ var textTypes = ['text'];
 
 module.exports = function(req, opts){
   req = req.req || req;
-  opts = opts || {};
```

Comment:
Yeah, I was just avoiding the allocation of and subsequent clone of (once it's passed into one of the types) an empty object. The checks to avoid it were minimal.